src/BizHawk.WinForms.Controls/BizHawk.WinForms.Controls.csproj

```diff
@@ -1,10 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>net48</TargetFramework>
+		<TargetFrameworks>net48;net6.0-windows</TargetFrameworks>
 	</PropertyGroup>
 	<Import Project="../MainSlnCommon.props" />
+	<PropertyGroup>
+		<UseWindowsForms>true</UseWindowsForms>
+		<EnableWindowsTargeting>true</EnableWindowsTargeting>
+	</PropertyGroup>

```

Comment:
This will break compilation on Linux, you'll need to do it conditionally

Reply:
I've specifically set `EnableWindowsTargeting` to *fix* compilation on linux. What compilation problems are you seeing? It compiles fine for me.

Reply:
I haven't tried. I recall from last time I messed with `hammer-and-duct-tape` that `<UseWindowsForms/>` didn't work w/ the `net48` TFM, and multi-targeting `net6.0-windows` prevented building on Linux—`<EnableWindowsTargeting/>` has obviously been added to fix that, but does it also fix the WinForms references?

Reply:
Well the net4.8 variant runs fine for me in WSL, so I assume everything is good on that front.
From the tests I've done it seemed that the `System.Windows.Forms` reference is functionally equivalent to setting `UseWindowsForms`.